library/avatar/avatar.css

```diff
@@ -0,0 +1,70 @@
+/*  Avatar Component Starts */
+.avatar{
+    width: 6.4rem;
+    height: 6.4rem;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+    color: var(--white);
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    position: relative;
+}
+.avatar__content {
+    width: 6.4rem;
+    height: 6.4rem;
+    border-radius: 50%;
+}
+
+/* Avatar Component Starts -- Extra Large */
```

Comment:
same: you can merge two different classes using a comma if they have the same styling

Reply:
Not Needed Anymore.